.github/workflows/rerun-pr-jobs.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     if: ${{ github.event.issue.pull_request }}
     runs-on: ubuntu-20.04
     steps:
-    - uses: estroz/rerun-actions@main
+    - uses: estroz/rerun-actions@v0.3.0
```

Comment:
Can we use a commit sha instead, when the action is sourced from an individual and not an organization, I think we should take the extra care here.

Reply:
Done.